<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Zafar" failures="0" tests="1" name="test3" time="3.821" errors="0" timestamp="2023-02-27T15:05:45 IST">
  <testcase classname="com.tyss.genricUtility.ConfigerClass.MainClass" name="@BeforeMethod loginApp" time="0.635">
    <failure type="org.openqa.selenium.StaleElementReferenceException" message="stale element reference: element is not attached to the page document
  (Session info: chrome=110.0.5481.105)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: &amp;apos;3.141.59&amp;apos;, revision: &amp;apos;e82be7d358&amp;apos;, time: &amp;apos;2018-11-14T08:17:03&amp;apos;
System info: host: &amp;apos;ZAFAR&amp;apos;, ip: &amp;apos;192.168.0.123&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;16.0.1&amp;apos;
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 110.0.5481.105, chrome: {chromedriverVersion: 110.0.5481.77 (65ed616c6e8e..., userDataDir: C:\Users\khanz\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:50290}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: d97c2eb91440e14163986d0a11a38fe8">
      <![CDATA[org.openqa.selenium.StaleElementReferenceException: stale element reference: element is not attached to the page document
  (Session info: chrome=110.0.5481.105)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/stale_element_reference.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'ZAFAR', ip: '192.168.0.123', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '16.0.1'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 110.0.5481.105, chrome: {chromedriverVersion: 110.0.5481.77 (65ed616c6e8e..., userDataDir: C:\Users\khanz\AppData\Loca...}, goog:chromeOptions: {debuggerAddress: localhost:50290}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: d97c2eb91440e14163986d0a11a38fe8
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebElement.execute(RemoteWebElement.java:285)
at org.openqa.selenium.remote.RemoteWebElement.click(RemoteWebElement.java:84)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:51)
at jdk.proxy2/jdk.proxy2.$Proxy16.click(Unknown Source)
at com.lims.objectRepository.elements.LimsLoginPage.loginAction(LimsLoginPage.java:45)
at com.tyss.genricUtility.ConfigerClass.MainClass.loginApp(MainClass.java:85)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod loginApp -->
  <testcase classname="com.tyss.genricUtility.ConfigerClass.MainClass" name="@AfterMethod LogoutApp" time="-1.677490544907E9">
    <skipped/>
  </testcase> <!-- @AfterMethod LogoutApp -->
  <testcase classname="com.tyss.genricUtility.ConfigerClass.MainClass" name="@BeforeMethod loginApp" time="0.654">
    <failure type="java.lang.AssertionError" message="Home Page is Open Successfully expected [Home] but found [Login Page]">
      <![CDATA[java.lang.AssertionError: Home Page is Open Successfully expected [Home] but found [Login Page]
at com.tyss.genricUtility.ConfigerClass.MainClass.loginApp(MainClass.java:86)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:831)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod loginApp -->
  <testcase classname="com.tyss.genricUtility.ConfigerClass.MainClass" name="@AfterMethod LogoutApp" time="-1.677490544918E9">
    <skipped/>
  </testcase> <!-- @AfterMethod LogoutApp -->
  <testcase classname="importantTestCase.Client_Add_Payment" name="client_Add_Payment" time="0.0">
    <skipped/>
  </testcase> <!-- client_Add_Payment -->
</testsuite> <!-- test3 -->
